app/src/main/java/me/saket/dank/ui/UrlRouter.java

```diff
@@ -228,12 +228,11 @@ public static String findAllowedPackageNameForDeepLink(String url) {
     }
   }
 
-  public static boolean isPackageNameInstalled(Context context, String packageName) {
+  public static boolean isPackageNameInstalledAndEnabled(Context context, String packageName) {
     PackageManager packageManager = context.getPackageManager();
     try {
       packageManager.getPackageInfo(packageName, PackageManager.GET_ACTIVITIES);
-      return true;
-
+      return packageManager.getApplicationInfo(packageName, 0).enabled;
```

Comment:
Isn't the line above that uses `getPackageInfo` now no longer necessary? `getApplicationInfo` should throw exception on its own if the package is not found.

Reply:
Good catch. Thanks
I will remove the line